pkg/virt-controller/watch/snapshot/restore.go

```diff
@@ -166,19 +161,30 @@ func (ctrl *VMRestoreController) updateVMRestore(vmRestoreIn *snapshotv1.Virtual
 					return 0, ctrl.doUpdateError(vmRestoreIn, err)
 				}
 
-				ctrl.Recorder.Eventf(
-					vmRestoreOut,
-					corev1.EventTypeNormal,
-					restoreCompleteEvent,
-					"Successfully completed VirtualMachineRestore %s",
-					vmRestoreOut.Name,
-				)
-
-				t := true
-				vmRestoreOut.Status.Complete = &t
-				vmRestoreOut.Status.RestoreTime = currentTime()
-				updateRestoreCondition(vmRestoreOut, newProgressingCondition(corev1.ConditionFalse, "Operation complete"))
-				updateRestoreCondition(vmRestoreOut, newReadyCondition(corev1.ConditionTrue, "Operation complete"))
+				updated, err := target.UpdateDoneRestore()
```

Comment:
this 'updated' name variable is a bit confusing cause it indicates if an update was required or not and not if the value is already updated or not, if you understand what I mean. maybe 'updateRequired' or something similar will be more clear